source/extensions/omni.isaac.lab/omni/isaac/lab/sim/schemas/schemas.py

```diff
@@ -44,7 +44,7 @@ def define_articulation_root_properties(
     """
     # obtain stage
     if stage is None:
-        stage = stage_utils.get_current_stage()
+        stage = omni.usd.get_context().get_stage()
```

Comment:
- Is the reason for not using the stage_utils.get_current_stage() just to reduce function calls and checks for fabric? 
- Is this a speed/performance boost? 
- Will we ever want to support fabric stages?

Reply:
When we want to move to fabric, we'll do it everywhere at once. Right now it seems an overkill to have that function for something that is weakly supported even in isaac-core side.